.github/workflows/static.yml

```diff
@@ -0,0 +1,57 @@
+name: Static analysis
+
+on:
+  pull_request: ~
+  push:
+    branches:
+      - master
+
+jobs:
+  phpstan:
+    name: PHPStan
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Set up PHP
+        uses: shivammathur/setup-php@2.7.0
+        with:
+          php-version: '7.4'
```

Comment:
Let's change this to whatever version we run on the production server

Reply:
7.2 at the moment - but we should maybe keep this and then upgrade to 7.4 https://github.com/carsonbot/carsonbot/blob/master/.symfony.cloud.yaml

Reply:
Okey. Cool cool cool